src/main/java/com/o19s/es/explore/ExplorerQuery.java

```diff
@@ -91,6 +92,10 @@ public int hashCode() {
     @Override
     public Weight createWeight(IndexSearcher searcher, ScoreMode scoreMode, float boost)
             throws IOException {
+        if (!LTRSettings.isLTRPluginEnabled()) {
+            throw new IllegalStateException("LTR plugin is disabled. To enable update ltr.plugin.enabled to true");
```

Comment:
```suggestion
            throw new IllegalStateException("LTR plugin is disabled. To enable, update ltr.plugin.enabled to true");
```
BTW, can you share where this setting defined `ltr.plugin.enabled? Can't find it.